src/WorkerChannel.ts

```diff
@@ -15,8 +15,14 @@ export class WorkerChannel {
     eventStream: IEventStream;
     functionLoader: IFunctionLoader;
     packageJson: PackageJson;
+    hostVersion = '';
+    // this hook data will be passed to (and set by) all hooks in all scopes

```

Comment:
```suggestion
    /**
     * this hook data will be passed to (and set by) all hooks in all scopes
     */
```

If you do comments like this, they'll show up in Intellisense. I don't know the details on why, but it's just always been that way 🤷‍♂️
<img width="1231" alt="Screen Shot 2022-06-27 at 6 18 06 PM" src="https://user-images.githubusercontent.com/11282622/176066295-1d53d4fb-884b-4029-83d1-462a61670d77.png">